### PR TITLE
tests: finish GitHub Actions CI setup

### PR DESCRIPTION
**Pull request by ploxiln**
follow-up to #1295

### CODE REVIEW

**Comment by ploxiln** [comment]
I think this is ready to go, but feel free to suggest changes @jehiah @mreiferson

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 </ul>
 </p>
 
-[![Build Status](https://secure.travis-ci.org/nsqio/nsq.svg?branch=master)](http://travis-ci.org/nsqio/nsq) [![GitHub release](https://img.shields.io/github/release/nsqio/nsq.svg)](https://github.com/nsqio/nsq/releases/latest) [![Coverage Status](https://coveralls.io/repos/github/nsqio/nsq/badge.svg?branch=master)](https://coveralls.io/github/nsqio/nsq?branch=master)
```
Comment (mreiferson) [inline README.md:13]: Let's replace this with GitHub Actions status badge? https://docs.github.com/en/free-pro-team@latest/actions/managing-workflow-runs/adding-a-workflow-status-badge

**Review by mreiferson — CHANGES_REQUESTED** [review]

**Comment by mreiferson** [comment]
I suppose one other question is: can we make the coveralls build be informational only?

File: README.md (hunk 1 above)
Comment (ploxiln) [reply README.md:13]: sure

**Comment by ploxiln** [comment]
I think you would make that change on the coveralls site, and I can't do that for nsqio/nsq, but you should see settings that look like this, and change the thresholds:

Screen Shot 2020-11-07 at 5 58 19 PM

**Pull request merged**